var/spack/repos/builtin/packages/intel-gtpin/package.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Reply:
I updated the copyright, sorry I missed that.

Reply:
Np. Thanks.